packages/webapp/pages/onboarding.tsx

```diff
@@ -440,7 +440,7 @@ export function OnboardPage(): ReactElement {
     <Container
       className={
         // Because of the video colours, we need to adjust the background colour a little bit
-        isOnboardingV3 ? 'bg-[#111119]' : 'bg-theme-bg-primary flex-col'
+        isOnboardingV3 ? 'bg-[#0e1019]' : 'bg-theme-bg-primary flex-col'
```

Comment:
This was intended?
Not sure it was only for testing the safari

Reply:
Yeah, intended it